web/client/components/data/identify/IdentifyContainer.jsx

```diff
@@ -106,14 +109,22 @@ module.exports = props => {
                         onChangeFormat={onChangeFormat}
                     /> || null,
                     <GeocodeViewer latlng={latlng} revGeocodeDisplayName={revGeocodeDisplayName} {...props}/>,
-                    buttons.length > 0 ? (
```

Comment:
you sure about this? if i remember correctly it was messing up the layout (try it with IE11)

Reply:
Checked, buttons array is never empty, so this check dosn't make sense